Tests/NIOTests/FixedLengthFrameDecoderTest.swift

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import NIO
+
+class FixedLengthFrameDecoderTest: XCTestCase {
+
+    public func testDecodeIfLessBytesAreSend() throws {
+        let channel = EmbeddedChannel()
+
+        let frameLength = 8
+        _ = try channel.pipeline.add(handler: FixedLengthFrameDecoder(frameLength: frameLength)).wait()
```

Comment:
remove the `_ =` that's not needed as the type should be `Void` so you don't need to ignore it.